docs/backbone.radio.md

```diff
@@ -85,7 +23,9 @@ access to this API.
   * [Listening to Requests on Objects](#listening-to-requests-on-objects)
   * [When to use Requests](#when-to-use-requests)
 
-## Channel
+## Radio Concepts
+
```

Comment:
Done.
I also updated the index and fixed the lint issues. By my side is ready to go